### PR TITLE
Added inital AF plans

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: af-plans/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Automation Framework Plans
+==========================
```
Comment (kingthorin) [inline af-plans/README.md:2]: Use markdown for the heading?

(This probably renders markdownishly but I mean using hash marks 😉)

File: af-plans/BaselineExample.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- parameters: {}
+  name: "spider"
+  type: "spider"
+  tests:
+  - onFail: "INFO"
+    statistic: "automation.spider.urls.added"
+    site: ""
+    operator: ">="
+    value: 100
+    type: "stats"
```
[the remark this answers is not in the document]
Comment (thc202) [reply af-plans/BaselineExample.yaml:45]: No, that's mentioned in the initial plan description.

[the remark this answers is not in the document]
Comment (kingthorin) [reply af-plans/BaselineExample.yaml:45]: Doh, skipped the documentation, my fault

File: af-plans/README.md (hunk 1 above)
Comment (psiinon) [reply af-plans/README.md:2]: I copied the other READMEs 😉 
I'd prefer to stick with consistancy...

Comment (kingthorin) [reply af-plans/README.md:2]: That’s fine I hadn’t checked

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**